Dockerfile.openvino

```diff
@@ -184,3 +179,10 @@ COPY . /mediapipe/
 ENV GLOG_logtostderr=1
 ENV LD_LIBRARY_PATH="/opt/intel/openvino/runtime/lib/intel64/:${LD_LIBRARY_PATH}"
 RUN bash build_desktop_examples.sh -b
```

Comment:
building examples should be started after all dependencies are installed

Reply:
Ok